classes/article/AuthorDAO.inc.php

```diff
@@ -120,9 +120,9 @@ function &getAuthorsAlphabetizedByJournal($journalId = null, $initial = null, $r
 				aa.middle_name,
 				aa.last_name,
 				CASE WHEN asl.setting_value = \'\' THEN NULL ELSE SUBSTRING(asl.setting_value FROM 1 FOR 255) END AS affiliation_l,
-				asl.locale,
+				CASE WHEN asl.setting_value = \'\' THEN NULL ELSE asl.locale END AS locale,
```

Comment:
Yes, I was also thinking so, but wenn asl.setting_value is NULL the locale and primary_locale are also however NULL :-) So do you still think I should consider it in the case statement?